packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1632,16 +1634,35 @@ class RenderTransform extends RenderProxyBox {
   ///
   /// This is equivalent to setting an origin based on the size of the box.
   /// If it is specified at the same time as an offset, both are applied.
-  Alignment get alignment => _alignment;
-  Alignment _alignment;
-  set alignment(Alignment value) {
-    assert(value == null || (value.x != null && value.y != null));
+  ///
+  /// A [AlignmentDirectional.start] value is the same as an [Alignment]
```

Comment:
An

Reply:
There were a lot of these, I think I've got them all.